server/admin-function/check-admin.ts

```diff
@@ -0,0 +1,21 @@
+"use server";
+
+import { dbAdmin } from "@/lib/firebase-admin";
+
+export const checkAdmin = async (domain: string, loggedInUserId:string): Promise<boolean> => {
```

Comment:
rename to userId: string and export const verifyUserAdmin

Reply:
good